src/olympia/landfill/management/commands/fetch_prod_addons.py

```diff
@@ -70,6 +74,10 @@ def _handle_addon(self, addon_data):
                 'platform': amo.PLATFORM_DICT[files[0]['platform']].id,
                 'size': files[0]['size'],
                 'is_webextension': files[0]['is_webextension'],
+                'is_mozilla_signed_extension': (
+                    files[0]['is_mozilla_signed_extension']),
+                'strict_compatibility': (
+                    version['is_strict_compatibility_enabled'])
             }
```

Comment:
Is there anything else we're not importing?  Maybe there could be a test that would break if we added something to the serializer and didn't added it to here/an exclusion list?

Reply:
A bit actually yeah. There is a TODO in the file to do license, ratings, previews, android compat data, category info. And well, since the stuff I added in that PR was not part of that TODO, I guess more is still missing ;) Ultimately though, everything we need to test search relevancy is already there now I think, at least for Firefox add-ons.